src/components/button-options/ButtonOptions.tsx

```diff
@@ -94,41 +87,17 @@ const ButtonOptions = ({ state }: { state: LocationState }) => {
         </Button>
       </ButtonGroup>
 
-      <Popper
-        sx={{
-          zindex: 1,
-        }}
-        open={openOptions}
-        anchorEl={buttonRef.current}
-        role={undefined}
-        transition
-        disablePortal
-      >
-        {({ TransitionProps, placement }) => (
-          <Grow
-            {...TransitionProps}
-            style={{
-              transformOrigin: placement === 'bottom' ? 'center top' : 'center bottom',
-            }}
+      <OptionListContainer buttonRef={buttonRef.current} handleClose={handleClose}>
+        {copyOptions.map((option, idx) => (
+          <MenuItem
+            key={option + idx}
```

Comment:
Is there a reason you used the key as option + idx?

Reply:
there's no way about using both, just thought that {option + idx} would distinguish each option better